test/lib/services/projectService.test.js

```diff
@@ -0,0 +1,57 @@
+describe('ProjectService', () => {
+    beforeEach(() => {
+        jest.resetModules();
+        console.log = jest.fn(); // create a new mock function for each test
+    })
+    test('createNewProjectPositive', async() => {
```

Comment:
rename to "should create new project successfully"

Reply:
also, add new space after async

Reply:
space like async () ?

Reply:
done